specification/authorization/resource-manager/Microsoft.Authorization/2015-07-01/authorization.json

```diff
@@ -491,7 +491,7 @@
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The ID of the role assignment to delete.",
+            "description": "The fully qualified ID of the role assignment, including the scope, resource name and resource type. Use the format, /{scope}/providers/Microsoft.Authorization/roleAssignments/{roleAssignmentName}. Example: /subscriptions/{subId}/resourcegroups/{rgname}//providers/Microsoft.Authorization/roleAssignments/{roleAssignmentName}.",
```

Comment:
>Use [](start = 129, length = 3)

Should we say Please use

Reply:
I don't think it is necessary. Example [here](https://github.com/Azure/azure-rest-api-specs/blob/current/specification/resources/resource-manager/Microsoft.Resources/2017-05-10/resources.json#L1373)